modules/apps/calendar/calendar-api/src/main/java/com/liferay/calendar/util/CalendarBookingIterator.java

```diff
@@ -107,7 +107,7 @@ private Calendar _getStartTimeJCalendar(DateValue dateValue) {
 			_getTimeZone(_calendarBooking));
 
 		int shift = JCalendarUtil.getDSTShift(
-			jCalendar, startTimeJCalendar, _displayTimeZone);
+			jCalendar, startTimeJCalendar, _getTimeZone(_calendarBooking));
```

Comment:
Hi @IstvanD,

could you replace `_getTimeZone`... for this:

`(_calendarBooking.isAllDay()) ? TimeZone.getTimeZone(StringPool.UTC) : _displayTimeZone`

Thanks!

Reply:
hi @MarinhoFeliphe thanks again for checking! I see that you replaced the method for the expression in your new pull request. I assume I don't have to do it at this point. Is that correct?

Reply:
Exactly, @IstvanD

I'm just trying to find the cause of the other errors, I'll keep you updated.

Thanks!